### ISSUE TITLE
duplicate argument

### ISSUE THREAD

**Issue by koo5** [oazapfts/oazapfts#58]
using https://raw.githubusercontent.com/prometheus/alertmanager/2b6315f399b95c4e327dcd5f5eb10866ef844252/api/v2/openapi.yaml

result:
```
export function deleteSilence(silenceId: string, silenceId: string, opts?: Oazapfts.RequestOpts) {
```

**Comment by Grahnz** [comment]
+1 I have experienced the same bug using inhouse API specs

**Comment by fgnass** [comment]
Closed by 3d4634c
